docs/features/vpn.rst

```diff
@@ -32,8 +32,9 @@ increase throughput, although in practice the gain is minimal.
 
 **Site configuration:**
 
-1) Install ``gluon-web-mesh-vpn-fastd`` in ``site.mk``
+1) Add the package ``gluon-web-mesh-vpn-fastd`` in ``site.mk``
 2) Set ``mesh_vpn.fastd.configurable = true`` in ``site.conf``
+3) optional add ``null`` to the ``mesh_vpn.fastd.methods`` table if you want "Performance mode" as default (not recommended)
```

Comment:
1) *optionally
2) is performance mode automatically the default, if you add it? your comment suggests this.

Reply:
done